execution_times.txt

```diff
@@ -0,0 +1,54 @@
+This file contains execution times of some functions to see what we should limit so the test code doesn't bloat
```

Comment:
We don't need to keep this file in the repository. Feel free to paste the contents in a PR comment, this way when referring back to your commits it will remain as a reference.

Reply:
Alright sure